src/cpp/src/speculative_decoding/speculative_decoding_impl.cpp

```diff
@@ -222,6 +222,7 @@ void ContinuousBatchingPipeline::SpeculativeDecodingImpl::step() {
         auto main_model_gen_duration = main_timer.get_duration_microsec();
         auto m_main_pipeline_metrics = m_main_pipeline->get_metrics();
         main_raw_perf_counters.m_durations.push_back(MicroSeconds(main_model_gen_duration));
+        // TODO: Ask about +=
         main_raw_perf_counters.m_inference_durations[0] = MicroSeconds(m_main_pipeline_metrics.inference_duration);
```

Comment:
Hello Dear @sbalandi!
Could you please say, do you think the `=` is a typo and `+=` should be used here?

Reply:
@AsyaPronina yes, it looks like typo
also here https://github.com/openvinotoolkit/openvino.genai/blob/master/src/cpp/src/speculative_decoding/continuous_batching_for_speculative_decoding_impl.cpp#L323